### PR TITLE
removed glide references and first attempt to pipeline checks

### PR DESCRIPTION
**Pull request by alejandroEsc**

### CODE REVIEW

**Comment by alejandroEsc** [comment]
@joejulian @leahnp @oneilcin @sostheim ready for review.

**Comment by sostheim** [comment]
@alejandroEsc - can you check the CI test failure please.

**Comment by oneilcin** [comment]
This LGTM.  I just have one question, why are all the checks failing?  Does this need a rebase?
On my PR 19 All checks passed.

**Comment by alejandroEsc** [comment]
@leahnp this was rebased. The checks passed before because not all the packages were checked. I fixed that here, now all code is being checked.

**Comment by oneilcin** [comment]
Ah golint and gocyclo errors in the controller

File: hack/common.sh [hunk 1]
```diff
@@ -22,8 +22,7 @@ function inf {
 }
 
 packages() {
-    // TODO
-  echo ""
+  echo "./ ./pkg/apis/clustercontroller ./pkg/apis/clustercontroller/v1alpha1 ./pkg/signals"
```
Comment (alejandroEsc) [inline hack/common.sh]: @leahnp here is the location where we write in where code to check lives.

**Comment by joejulian** [comment]
Is there anything that can be done within this PR to make the gocyclo output meaningful? I have no idea what this is trying to tell us. Maybe there's a flag that needs set or something?
```
Verifying /home/jenkins/workspace/cluster-controller_PR-18-DIFLIA6PJDWNKBYRQVEWMBIIDQRN43HBWVUR4EXER6XAREXXMHDQ/hack/../hack/checks/verify-gocyclo.sh

17 main (*Controller).syncHandler main.go:147:1

FAILED   /home/jenkins/workspace/cluster-controller_PR-18-DIFLIA6PJDWNKBYRQVEWMBIIDQRN43HBWVUR4EXER6XAREXXMHDQ/hack/../hack/checks/verify-gocyclo.sh	0s
```

**Comment by alejandroEsc** [comment]
@joejulian you can check that locally, but i can figure that out later.

**Comment by joejulian** [comment]
Bummer.

**Comment by joejulian** [comment]
Approved. :) My only gripe is that if the description has to use the word "and" it should probably be multiple PRs.

**Pull request merged**